src/server.js

```diff
@@ -42,17 +42,20 @@ module.exports = function() {
 		server = require("http");
 		server = server.createServer(app);
 	} else {
-		server = require("spdy");
```

Comment:
Oh, so you did. I missed that. The green on github is sometimes really easy to miss.